.mention-bot

```diff
@@ -1,9 +0,0 @@
-{
-  "maxReviewers": 2,
```

Comment:
I know we used to have it, it used to say something like:

> By analizing your changes, we found <user> and <user> to be potential reviewers

Or something like that. The thing, I was not aware it was removed/not working, which means I did not miss it, so it can be removed for me, but I'd like to be sure of it :P